README.md

```diff
@@ -1 +1,126 @@
+
+Key Objectives:
+- Enhanced Search Capability: The app facilitates users in searching for parking spots near their desired locations, empowering them to plan their trips more efficiently.
+
+- Tailored Filters for Accessibility: Recognizing the diverse needs of individuals, the app offers filters allowing users to customize search results based on radius, fees, and the availability of disabled parking spaces.
+
+- User-Centric Experience: By providing a history of previous searches, the app ensures that users can effortlessly revisit past locations, fostering a more user-centric and personalized experience.
+
+- Detailed Information and Feedback: Each search result is enriched with images, detailed descriptions, and user feedback, offering a comprehensive view of the parking spot's accessibility.
+
+In the process of building this app, we learned invaluable lessons about the intersection of technology and accessibility. We gained insights into designing user interfaces that prioritize inclusivity, utilizing APIs such as Google Maps to enhance location-based services as well as Open Weather API to help with creating a 360 view of what the users destination will be like and creating a seamless user experience for individuals with diverse needs.
+
+The Parking Finder App is not just a project; it's a commitment to making urban mobility more accessible and enjoyable for everyone.
+
+
+## Features
+
+1. Search Feature
+User Story
+As a person with a car, I want to be able to search for a destination and view parking spots near it, so that I can plan my trip better.
+
+Acceptance Criteria
+Upon opening the application, users can search for their desired location.
+The map displays results based on the search, allowing users to interact with the map.
+
+2. Filter Feature
+User Story
+As a person with special needs, I want to be able to filter parking spots so that I can find a spot that matches my requirements.
+
+Acceptance Criteria
+Users are presented with search results.
+Filtering options include criteria such as radius, fees, and disabled parking availability.
+
+3. Search History Feature
+User Story
+As a person using the app, I want to see my search history so that I can know where I have searched previously.
+
+Acceptance Criteria
+Clicking on 'View Previous Searches' allows users to navigate through their search history.
+Previous search results are accessible for review.
+
+4. Display Description Feature
+User Story
+As a person with a car, I want to select from a filtered search result, so that I can see feedback, images, and descriptions of parking facilities.
+
+Acceptance Criteria
+Clicking on a specific result provides users with images and detailed descriptions of that particular location.
+Users can leave feedback on whether the parking spot was accessible.
+
+
+
```

Comment:
Instead of key objectives, we should do key features only, they are kind of overlapping. and also I don't think we should include all of the user stories and acceptance criteria in here, they make the readme file too long and they are also for internal usage as well